src/OpenTK.Windowing/NativeWindow.cs

```diff
@@ -39,13 +40,7 @@ public string Title
 
         public bool Focused
         {
-            get
-            {
-                unsafe
-                {
-                    return glfw.GetWindowAttrib(windowPtr, (int)WindowHint.GLFW_FOCUSED) != 0;
-                }
-            }
+            get { unsafe { return glfw.GetWindowAttrib(windowPtr, (int)WindowHint.GLFW_FOCUSED) != 0; } }
```

Comment:
why not like this?
`unsafe get => glfw.GetWindowAttrib(windowPtr, (int)WindowHint.GLFW_FOCUSED) != 0;`
Or even use it on the whole property?
`public unsafe bool Focused`

Should make it a lot more readable in my opinion, apart from that it has no deficit as far as I can tell

Reply:
@jvbsl To answer your first suggestion
![unsafe_accessor_invalid](https://user-images.githubusercontent.com/2267600/50742422-ebe87900-11d8-11e9-927e-a2ce550ed52b.png)

I have applied unsafe to the property instead and updated the get accessor to an expression body.
I don't know why I was under the assumption this would expose something unnecessary. Thanks!

Reply:
that is really strange as, I tried it out with simple Code inside Rider(.Net Core 2.1), just to be sure that it works, because I actually wasn't...But it would still make sense, at it is just another method....
Apart from that the second approach I mentioned should still work though?

Reply:
Fixed quite a while ago